OrderService/src/main/resources/application.properties

```diff
@@ -1,6 +1,10 @@
 spring.application.name=OrderService
+server.port=8081
 
 management.endpoints.web.exposure.include=health
 
-server.port=8081
+spring.data.mongodb.uri=mongodb://localhost:27017/order_db
```

Comment:
NIT COMMENT:- Think of a way to have this dynamic, We can not hardcode the DB details in the code. This might change later.